### ISSUE TITLE
3.8.0 - data on activeUser not consistent structure between user functions

### ISSUE THREAD

**Issue by alexbainbridge** [Kinvey/html5-sdk#22]
#### Steps to Reproduce
Since 3.8.0

The response to Kinvey.initialize (which resolves the active user) is not the same response as Kinvey.User.getActiveUser() (which also should return the active user)

#### Expected Result
Should match

They used to match on previous version of the library

#### Actual Result
The data about the user is no longer in "data", but moved up a level, on the Kinvey.initialize response. Data is still there of course, just not positioned where it used to be, structurally.

#### Version of Kinvey HTML5 SDK
3.8.0

Not a major crisis but when I updated the library and found our app failed completely, was a bit of a shock until I could see what the cause was.

**Comment by alexbainbridge** [comment]
Did you spot this one @thomasconner ?  I can explain further if you wish.  

Putting our app (next release) live using 3.8.0 tonight, and coded to handle how you have changed it, so leaving this ticket should you choose to put your active user response back to consistent response from the two functions mentioned previously.

All good though, keep iterating ;)

**Comment by thomasconner** [comment]
@alexbainbridge This makes sense and you are correct the structure did change. We moved it up a level because we were storing the client along with the active user data. We did not intentionally mean to store the client but only the active user data.

We will look into returning an instance of the `Kinvey.User` with `Kinvey.initialize()` so the result is the same that you would get from `Kinvey.User.getActiveUser()`. In the meantime, you can just call `Kinvey.User.getActiveUser()` to prevent from coding for the different structure. See the example code snippet below.

```javascript
Kinvey.initialize({
  appKey: '<appKey>',
  appSecret: '<appSecret>'
})
  .then(function() {
    var activeUser = Kinvey.User.getActiveUser();
  });
```

_P.S. - I just wanted to point out we deprecated `Kinvey.initialize()'. You might already be aware of this but if not I wanted to make sure you knew._

We will certainly keep iterating and we have a lot of stuff in the pipeline. I appreciate the kind words and enjoy your involvement with helping us continually improve the SDKs.

**Comment by alexbainbridge** [comment]
That approach is 2 API calls though yes? When previous concept, using a response, was 1. With mobile would prefer 1 round trip vs 2, if possible?

Fixed it anyways by accessing the data differently, so no big problem. (although broke a lot this end until I worked it out)

re initialize depreciation..... I checked the docs and it now uses init - and that doesnt work (by just changing initialize to init) - so slightly confused. Mind you, it is Friday night. I am using it pretty much as per your snippet above, initialize. 

Thanks!
Alex
